src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -5415,6 +5415,13 @@ def test_aks_automatic_sku_with_hosted_system_enabled(self, resource_group, reso
             self.is_empty(),
         ])
 
+        # verify that hosted system profile is enabled and agent pool profiles are None/empty
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=141648639&view=results) to validate the change.